tests/strictness-checks/Orphans.hs

```diff
@@ -0,0 +1,16 @@
+{-# options_ghc -Wno-orphans #-}
+
+module Orphans where
+
+import qualified Data.List.NonEmpty as NonEmpty
+import Data.List.NonEmpty (NonEmpty ((:|)))
+import Data.Maybe
+import Test.Tasty.QuickCheck
+
+instance Arbitrary a => Arbitrary (NonEmpty a) where
```

Comment:
I understand the motivation behind this module, but let's avoid orphans. For instance, in future it can clash with `quickcheck-instances` badly. If you really need such instance, feel free to define `newtype MyNonEmpty a = MyNonEmpty (NonEmpty a)` and provide `instance Arbitrary (MyNonEmpty a)`.

Reply:
Yea, imma do that.

Reply:
Done.

Reply:
Disaster, it is so ugly. So much syntactic noise for a glorified tuple. First they add a type to `base` and then I cannot really use it even a dozen releases later.

Maybe I should actually make it a tuple instead.

Reply:
Imma keep it ugly for now.